src/vcard.c

```diff
@@ -40,6 +40,82 @@
 #include "mem.h"
 #include "vcard.h"
 
+/**
+ * Compile regex, checking and handling errors.
+ *
+ * \parm[in] preg The compiled regex.
```

Comment:
I guess the doxygen should be `out`, not `in` for this parameter!
By the way, have you ever attempted a doxygen run over this source? I haven't!